ChangeLog

```diff
@@ -488,6 +488,8 @@ New deprecations
      deprecated.
    * PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH and PSA_ALG_AEAD_WITH_TAG_LENGTH
      have been renamed, and the old names deprecated.
+   * Sign-magnitude and one's complement representations for signed integers are
```

Comment:
This should be in the section “Requirement changes”, not “New deprecations”. Something that's deprecated is still supported. But we're saying that as of now, this is not supported at all.